apps/web/components/ui/Text/Largetitle/Largetitle.tsx

```diff
@@ -1,14 +1,13 @@
-import classnames from "classnames";
 import React from "react";
+import { twMerge } from "tailwind-merge";
 
 import { TextProps } from "../Text";
 
 const Largetitle: React.FunctionComponent<TextProps> = (props: TextProps) => {
-  const classes = classnames(
-    "font-cal tracking-wider text-3xl text-gray-900 dark:text-white mb-2",
+  const classes = twMerge(
```

Comment:
what does twMerge do differently to classNames? tailwind labs themselves use classNames

Reply:
it resolves conflicting tailwind classes, which is the root for the issue of this fix
default tailwind classes are the ones you pass through, second parameter overrides default ones, so if a default text color is present and you pass another one, the second one gets through and the first one vanishes